src/name_bazaar/ui/spec.cljs

```diff
@@ -119,6 +119,17 @@
 
 (s/def ::now date?)
 
+(s/def :public-resolver.record/addr address?)
+(s/def :public-resolver/record
+  (s/keys :opt [:public-resolver.record/addr]))
+
+(s/def :resolver-records/entry (s/map-of :ens.record/node :public-resolver/record))
+
+(s/def :public-resolver/records
+  (s/keys :opt [:resolver-records/entry]))
+
+(s/def ::public-resolvers (s/map-of address? :public-resolver/records))
```

Comment:
You know what, let's just ditch public-resolvers addresses completely. We'll ignore them for reverse resolution as well. 99% of names will use just that anyway. So put default public resolver address `0x5ffc014343cd971b7eb70732021e26c35b744cc4` into `name-bazaar.ui.constants` and use it like that. 
`:public-resolver/records` will be at root of db, `::public-resolvers`. 

You can also skip loading name resolver address in other part of code, and just directly query default resolver.

Reply:
Or actually not into `name-bazaar.ui.constants` sorry, you can put it into `name-bazaar.shared.smart-contracts` for `:public-resolver`